locale/hu.json

```diff
@@ -31,256 +31,256 @@
         "status_offline": "**%{servername}** jelenleg **Nem Elérhető**!"
     },
     "nui_warning": {
-        "title": "FIGYELEM",
-        "warned_by": "Figyelmeztet:",
-        "instruction": "Tartsd nyomva a [SPACE]-t 10 másodpercig, hogy eltüntesd az üzenetet."
+        "title": "FIGYELMEZTETÉS",
+        "warned_by": "Figyelmeztetett:",
+        "instruction": "Tartsd lenyomva a [SPACE]-t 10 másodpercig, hogy eltüntesd az üzenetet."
     },
     "nui_menu": {
         "misc": {
-            "help_message": "txAdmin Menu enabled, type /tx to open it.\nYou can also configure a keybind at [Game Settings > Key Bindings > FiveM > Menu: Open Main Page].",
-            "menu_not_allowed": "Your identifiers do not match any admin registered on txAdmin.\nIf you are registered on txAdmin, go to Admin Manager and make sure your identifiers are saved.",
-            "action_unauthorized": "You do not have this permission.",
-            "unknown_error": "An unknown error occurred.",
-            "not_enabled": "The txAdmin Menu is not enabled! You can enable it in the txAdmin settings page.",
-            "announcement_title": "Server Announcement by %{author}:",
-            "general_no_perms": "You do not have permissions for this!",
-            "dialog_empty_input": "You cannot have an empty input."
+            "help_message": "A txAdmin rendelkezik játékon bellüli menüvel!\\\\nGyorsgomb [Game Settings > Key Bindings > FiveM > Open the txAdmin Menu] Próbáld ki!",
+            "menu_not_allowed": "Az azonosítód nem egyezik a txAdminban található admin azonosítókkal.\nHa regisztrálva vagy txAdminban, bizonyosodj meg róla hogy el van mentve az Admin Managerben.",
+            "action_unauthorized": "Ehhez nincs engedélyed.",
+            "unknown_error": "Váratlan hiba lépett fel.",
```

Comment:
Maybe `váratlan` could be replaced with `ismeretlen`, it would make more sense in my opinion.